ngraph_onnx/onnx_importer/utils/conv.py

```diff
@@ -143,24 +144,49 @@ def make_convolution_op(onnx_node, ng_inputs):
     :return: ngraph Op for convolution or deconvolution
     """
     if len(ng_inputs) == 3:
-        x, weights, bias = ng_inputs
+        data, weights, bias = ng_inputs
     elif len(ng_inputs) == 2:
-        x, weights = ng_inputs
-        bias = ng.constant(0, dtype=get_dtype(x.get_element_type()))
+        data, weights = ng_inputs
+        bias = ng.constant(0, dtype=get_dtype(data.get_element_type()))
     else:
         raise ValueError('Conv node (%s): unexpected number of input values: %d.',
                          onnx_node.name, len(ng_inputs))
 
     groups = onnx_node.get_attribute_value('group', 1)
-    if groups != 1:
-        log.warning('Conv node (%s): `group` attribute value %d is not supported.',
-                    onnx_node.name, groups)
 
     strides = get_strides(onnx_node)
     dilation = get_dilations(onnx_node)
     padding_below, padding_above = get_pads(onnx_node)
+    # reference code: https://github.com/NervanaSystems/ngraph-mxnet/blob/fdd692/src/ngraph/ngraph_emitter.cc#L822-L856
+    if groups != 1:
```

Comment:
Move reference code link inside this `if` statement and add a sentence explaining what's going on here.